dbms/programs/server/PrometheusMetricsWriter.cpp

```diff
@@ -20,6 +22,11 @@ void writeOutLine(DB::WriteBuffer & wb, T && val, TArgs &&... args)
     writeOutLine(wb, std::forward<TArgs>(args)...);
 }
 
+void replaceInvalidChars(std::string & metric_name)
```

Comment:
FWIW: normalize looks better

Reply:
As long as it is a function in .cpp file with hidden linkage, I don't care too much :)